### PR TITLE
fix(deps): Upgrade Less to v3

### PR DESCRIPTION
**Pull request by mengtzu**
`npm audit` raises some prototype pollution errors that trace back to dependencies of dependencies of our dusty old version of less.  Bumping to 3 appears to be non-traumatic.

### CODE REVIEW

**Comment by mengtzu** [comment]
I did find `fs` was failing in a handful of tests, so I switched the reference to `fs-extra` since sku uses that and I assume it'll be more robust (ie if `fs-extra` is not working neither will sku in general so 🤷‍♀️ )

File: test/test-cases/braid-design-system/__snapshots__/braid-design-system.test.js.snap [hunk 1]
```diff
@@ -26,8 +26,8 @@ Object {
 ._3Zn-PeU{border-radius:4px}
 ._1sseFqJ{border-style:solid;border-width:1px}
 ._1d6RKtE{-webkit-box-shadow:0 0 0 1px #2765cf;box-shadow:0 0 0 1px #2765cf}
-.mT50rPc{color:#1c1c1c}._1Rsg9w6{color:#fff}._3OSCp5v{color:#e60278}.AsBZy5a{color:green}._1DYv5gu{color:#1c1c1ca1 a1}._3rwI692{color:#2765cf}
-._16w04Xz{fill:currentColor}._1srDitw{fill:#2765cf}.hK-SldW{fill:#ccc}._3SkkQl1{fill:#e60278}.q2J3ENx{fill:green}._2WNYdGY{fill:#1c1c1ca1 a1}.QR_XQhw{fill:#fff}
+.mT50rPc{color:#1c1c1c}._1Rsg9w6{color:#fff}._3OSCp5v{color:#e60278}.AsBZy5a{color:green}._1DYv5gu{color:#1c1c1ca1}._3rwI692{color:#2765cf}
```
[the remark this answers is not in the document]
Comment (markdalgleish) [reply test/test-cases/braid-design-system/__snapshots__/braid-design-system.test.js.snap]: First glance, this looks like a broken test.

[the remark this answers is not in the document]
Comment (mengtzu) [reply test/test-cases/braid-design-system/__snapshots__/braid-design-system.test.js.snap]: Looking at the original snap it appears less was repeating the opacity, maybe as some kind of compatibility issue?  I don't think that's a problem if they've dropped that behaviour 🤔

[the remark this answers is not in the document]
Comment (markdalgleish) [reply test/test-cases/braid-design-system/__snapshots__/braid-design-system.test.js.snap]: Oh, sorry—didn't realise the summary version wasn't showing me the full diff. It looked like an entire line of CSS had been removed. I see what you mean now 👍

**Comment by mbfisher** [comment]
Awesome, those warnings have been bugging me!

File: test/test-cases/braid-design-system/__snapshots__/braid-design-system.test.js.snap (hunk 1 above)
[the remark this answers is not in the document]
Comment (michaeltaranto) [reply test/test-cases/braid-design-system/__snapshots__/braid-design-system.test.js.snap]: This seems to be the result of `v3.8.1` adding [support for #RRGGBBAA](https://github.com/less/less.js/pull/3291).  Highlights an interesting scenario though, snapshots here can change based on internal `braid-design-system` changes.

**Comment by mengtzu** [comment]
@mbfisher credit goes to Chia Lun in the Bricks team in Penang who was also bothered and reached out :)

**Pull request merged**